src/util/metaData.ts

```diff
@@ -18,18 +21,29 @@ ajv.addKeyword({
   },
 })
 
+ajv.addKeyword({
+  keyword: "uMaxLength",
+  type: "string",
+  validate: (max_length: number, value) => {
+     return value.split(byGrapheme).length <= max_length
+  },
+  error: {
+    message: (cxt: KeywordCxt): string => { return `must NOT have more than ${cxt.schemaValue} graphemes` },
+  }
+})
+
 const schema = {
   type: 'object',
   properties: {
     Name: { type: 'string', minLength: 1 },
     Creator: { type: 'string', minLength: 1 },
-    Description: { type: 'string', maxLength: 80, minLength: 1 },
+    Description: { type: 'string', uMaxLength: 80, minLength: 1 },
     Language: { type: 'string', minLength: 1, pattern: '^\\w{3}(,\\w{3})*$' },
     Publisher: { type: 'string', minLength: 1 },
-    Title: { type: 'string', maxLength: 30, minLength: 1 },
-    Date: { type: 'string', maxLength: 10, minLength: 10 },
+    Title: { type: 'string', uMaxLength: 30, minLength: 1 },
+    Date: { type: 'string', uMaxLength: 10, minLength: 10 },
```

Comment:
I think that for the `Date` field (which should be ASCII) there is no need to measure length by grapheme count

Reply:
yes, fixed